notebooks/official/automl/sdk_automl_tabular_regression_batch_bq.ipynb

```diff
@@ -0,0 +1,968 @@
+{
```

Comment:
<p>Use <code>TabularDataset.create()</code> to create a <code>Dataset</code> resource, which takes the following parameters: </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/331/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
<p>Done.</p>